docker/dbeaver/config/.cloudbeaver.runtime.conf

```diff
@@ -0,0 +1,47 @@
+{
+  "server": {
+    "rootURI": "/dbeaver/",
+    "expireSessionAfterPeriod": 1800000,
+    "database": {
+      "pool": {
+        "minIdleConnections": 4.0,
+        "maxIdleConnections": 10.0,
+        "maxConnections": 100.0,
+        "validationQuery": "SELECT 1"
+      },
+      "createDatabase": true,
+      "initialDataConfiguration": "conf/initial-data.conf",
+      "driver": "h2_embedded",
+      "url": "jdbc:h2:/opt/cloudbeaver/workspace/.data/cb.h2.dat"
```

Comment:
What's this h2 part for..? We only use sqlite for now, so is this right/needed..?

Reply:
I see, makes sense.